cms/static/js/views/utils/xblock_utils.js

```diff
@@ -86,12 +86,14 @@ define(["jquery", "underscore", "gettext", "common/js/components/utils/view_util
                         }
                     );
                 },
-                messageBody = interpolate(
+                messageBody;
+            xblockType = xblockType || 'component';
```

Comment:
While debugging the test, I notice this bug. xblockType was undefined at the time it was referred to.

![image](https://cloud.githubusercontent.com/assets/484484/15635913/15eba148-25bc-11e6-9364-a687177b9523.png)